server/client.go

```diff
@@ -932,6 +1045,11 @@ func (c *client) processMsg(msg []byte) {
 	}
 }
 
+func (c *client) pubPermissionViolation(subject []byte) {
+	c.sendErr(fmt.Sprintf("Permissions Violation for Publish to %q", subject))
+	c.Errorf("Publish Violation - User %q, Subject %q", c.opts.Username, subject)
```

Comment:
What I think we do is leave as an error, but I will mark a TODO to have thresholds where we cut off the client and/or blacklist them.